workflows/iuf/operations/extract-release-distributions.yaml

```diff
@@ -49,8 +49,15 @@ spec:
           productName="{{=jsonpath(inputs.parameters.global_params, '$.product_manifest.current_product.name')}}"
           echo "Extract from: $tarfile for product: $productName"
           dir=$(echo "$tarfile" | sed 's|\(.*\)/.*|\1|')
+          output=${tarfile%.tar.*}
+          echo ${output} > /tmp/parent_directory
           cd $dir
           tar -xvzf $tarfile
         volumeMounts:
           - name: iuf
             mountPath: /etc/iuf
+      outputs:
+        parameters:
+          - name: parent_directory
```

Comment:
I also wonder if `parent_directory` is the best name for this or not. Kinda sounds like the parent of the release distribution directory when really it's the extracted release distribution directory. Maybe we can change later if it's confusing.

Reply:
agreed, i am just taking whatever is in the scheme def so it won't break operations. we can talk about this monday